skyxplore-ui-test/src/test/java/com/mentoring/pageobject/FactoryPage.java

```diff
@@ -1,12 +1,24 @@
 package com.mentoring.pageobject;
 
 import com.mentoring.config.Config;
+import io.qameta.allure.Step;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
 public class FactoryPage extends Page {
 
     private static final String PAGE_PATH = "/equipment";
 
+    @FindBy(xpath = "//div[contains(text(),'Bővítő')]")
+    private WebElement expansionItem;
+
+    @FindBy(xpath = "//div[contains(@title,'CEX-01')]//button[contains(text(),'Gyártás indítása')]")
```

Comment:
No need for button label here, this is enough: //div[contains(@title,'CEX-01')]//button

Reply:
Fixed, thx!